src/components/DefaultBody.jsx

```diff
@@ -91,12 +91,12 @@ const SliderBody = ({
           </div>
         </Message>
       )}
-      {href && (
+      {(href || image) && (
```

Comment:
The condition is confusing, although meaning the same as the above one, it's not the same negated. Could we unify them? Or use a ternary.